godump.go

```diff
@@ -26,3 +26,20 @@ func DumpNC(v any) error {
 	}
 	return nil
 }
+
+// Sdump is just like Dump but returns the result instead of prining to STDOUT
```

Comment:
typo printing

Reply:
Thanks, this was resolved in [pull#5](https://github.com/yassinebenaid/godump/pull/5) and [pull#6](https://github.com/yassinebenaid/godump/pull/6)